scss/_patterns_chip.scss

```diff
@@ -19,6 +19,7 @@
 
     .p-chip__lead,
     .p-chip__value {
+      color: $colors--light-theme--text-default;
```

Comment:
Would it be better to set this `color` in the `vf-chip-type` mixin when the background-color is set?

Reply:
I thought as the text colour is always going to be the same and it isn't dynamic it made sense to set it with the rest of the text styles? But happy to change it just want to understand why

Reply:
Probably fine but my thought is that we might have variants that require adaptive colour based on the background.

Reply:
Currently all have same color, but when we introduce light/dark themes we will move the color setting to themes.

I think it's fine for now.